zebra/rt_netlink.h

```diff
@@ -45,22 +45,22 @@ extern "C" {
  * know about this new route.
  * tools/etc/iproute2/rt_protos.d
  */
-#define RTPROT_BGP         186
-#define RTPROT_ISIS        187
-#define RTPROT_OSPF        188
-#define RTPROT_RIP         189
-#define RTPROT_RIPNG       190
+#define RTPROT_BGP 186
```

Comment:
Avoid changing the formatting here if possible and only touch code you are changing